docs: use a consistent intro in the DB guides (#29797)
gravitational/teleport: docs/pages/includes/database-access/db-introduction.mdx

> Access to {{ dbName }} can be provided by [Teleport Database
> Access](../../database-access/introduction.mdx). This allows for
> fine-grain access control through [Teleport's RBAC](../../database-access/rbac.mdx).

Reviewer: ```suggestion
Teleport can provide secure access to {{ dbName }} via the  [Teleport Database
Service](../../database-access/introduction.mdx). This allows for
fine-grained access control through [Teleport's RBAC](../../database-access/rbac.mdx).
```
Since we're moving away from positioning each protocol as a separate product.
